packages/mermaid/src/mermaid.ts

```diff
@@ -337,6 +337,26 @@ const parse = async (text: string, parseOptions?: ParseOptions): Promise<boolean
   });
 };
 
+/**
+ * Function that renders an svg with a graph from a chart definition. Usage example below.
+ *
+ * ```javascript
+ *  element = document.querySelector('#graphDiv');
+ *  const graphDefinition = 'graph TB\na-->b';
+ *  const { svg, bindFunctions } = await mermaid.render('graphDiv', graphDefinition);
+ *  element.innerHTML = svg;
+ *  bindFunctions?.(element);
+ * ```
+ *
```

Comment:
Should we document this behavior in the function docs?:

```suggestion
 *
 * @remarks
 * Multiple calls to this function will be enqueued to run serially.
 *
```